src/pages/api/fetch-datasets.ts

```diff
@@ -0,0 +1,14 @@
+import type {NextApiRequest, NextApiResponse} from 'next';
```

Comment:
Would it be an option to use RESTful file names for the API endpoint (instead of RPC names)? For example: `src/pages/api/datasets.ts`?

Reply:
Good point